modules/library/src/main/scala/zio/elasticsearch/ElasticResult.scala

```diff
@@ -79,7 +71,8 @@ final class SearchResult private[elasticsearch] (private val hits: List[Item]) e
 final class SearchWithAggregationsResult private[elasticsearch] (
   private val hits: List[Item],
   private val aggregationsJson: Option[Json]
-) extends DocumentsWithAggregationsResult {
+) extends DocumentResult[List]
+    with AggregationsResult {
```

Comment:
Can we have `extends DocumentResult[List] with AggregationsResult`? In one line?

Reply:
`Scalafmt` does not allow it.